packages/react/docs/staticPages/component.example.jsx

```diff
@@ -0,0 +1,7 @@
+import {Metafield} from '@shopify/hydrogen-react';
```

Comment:
We don't have a metafield component anymore do we?

Reply:
oh.. you're right hahaha